packages/electrode-archetype-react-app/arch-clap.js

```diff
@@ -755,23 +755,24 @@ Individual .babelrc files were generated for you in src/client and src/server
 
     ".jest.test-frontend-cov": () => {
       const testDir = ["_test_", "__tests__"].find(x => shell.test("-d", x));
-
-      const srcJestFiles =
-        !testDir &&
-        glob.sync(`**/*.{test,spec}.{js,jsx,ts,tsx}`, {
+      let runJest = testDir;
+      if (!runJest) {
+        runJest = glob.sync(`**/*.{test,spec}.{js,jsx,ts,tsx}`, {
           cwd: Path.resolve(AppMode.src.dir)
-        });
-
-      if (testDir || srcJestFiles.length > 0) {
-        if (testDir) {
-          makeBabelRc(Path.join(testDir, "client"), "babelrc-client.js");
-        }
+        }).length > 0;
+      }
+      if (runJest) {
+        const { roots } = archetype.jest;
+        runJest = roots && roots.map(x => x.replace("<rootDir>", process.cwd()));
```

Comment:
what's the `replace` for?

Reply:
more specifically, what's using map/replace to process the array for?  The value of that array is not used anywhere.